lib/src/progress_bar.dart

```diff
@@ -81,7 +83,15 @@ class _VideoProgressBarState extends State<VideoProgressBar> {
         if (!controller.value.isInitialized) {
           return;
         }
-        _seekToRelativePosition(details.globalPosition);
+        if (Platform.isAndroid) {
+          // on android, we need to let the player buffer when scrolling
+          // in order to let the player buffer. https://github.com/flutter/flutter/issues/101409
+          if (!controller.value.isBuffering) {
+            _seekToRelativePosition(details.globalPosition);
+          }
+        } else {
+          _seekToRelativePosition(details.globalPosition);
+        }
```

Comment:
This sounds a bit suspect. Correct me if I'm wrong but the above says that if its running on Android and the `VideoPlayerController` is not buffering, then seek to its relative position. Otherwise, do the latter. 

Just wondering if that makes sense? Shouldn't it be:

```dart
        if (Platform.isAndroid && !controller.value.isBuffering) {
            _seekToRelativePosition(details.globalPosition);
        } else {
            _seekToRelativePosition(details.globalPosition);
        }
```

I think this might need to be cleaned up.

Reply:
added suggestion